client/components/payment-methods-list/payment-method.scss

```
@@ -43,17 +45,48 @@
 				cursor: not-allowed;
 			}
 		}
+
+		@include breakpoint( '<660px' ) {
+			align-self: flex-start;
+			margin-top: 5px;
+		}
+	}
+
+	&__text-container {
+		display: flex;
+		flex: 1;
+
+		@include breakpoint( '<660px' ) {
+			flex-wrap: wrap;
+		}
 	}
 
 	&__label {
-		display: inline-block;
 		font-size: 14px;
 		font-weight: 600;
 		line-height: 20px;
 		color: $gray-900;
 		margin-bottom: 4px;
 	}
 
+	&__label-desktop {
+		display: block;
+
+		@include breakpoint( '<660px' ) {
+			display: none;
+		}
+	}
+
+	&__label-mob {
```

May use `mobile`, which is cleared than `mob`? Also, update relevant `.scss` file too.

```suggestion
	&__label-mobile {
```